mmscan/__init__.py

```diff
@@ -0,0 +1,16 @@
+# flake8: noqa
+from mmscan.mmscan import MMScan
+
+print('MMScan module loaded')
+try:
+    from mmscan.evaluator.vg_evaluation import VG_Evaluator
```

Comment:
naming convention, we would like prefer the class name like VG_Evaluator -> VisualGroundingEvaluator
Please also fix other names